lib/qualtrics_api/extensions/serializable_collection.rb

```diff
@@ -0,0 +1,18 @@
+module QualtricsAPI
+  module Extensions
+    module SerializableCollection
+
+      def as_json
```

Comment:
probably we should allow ```as_json(options={})``` even if we're not using it to be consistent with to_json / as_json